pkg/keypair/secp256k1_test.go

```diff
@@ -0,0 +1,44 @@
+package keypair
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/iotexproject/iotex-core/pkg/hash"
+)
+
+func TestSecp256k1(t *testing.T) {
+	require := require.New(t)
+
+	k1, err := NewSecp256k1PrvKey()
+	require.NoError(err)
+	k2, err := NewSecp256k1PrvKey()
+	require.NoError(err)
+	k3, err := NewSecp256k1PrvKey()
+	require.NoError(err)
+	k4, err := NewSecp256k1PrvKey()
+	require.NoError(err)
+
+	tests := []PrivateKey{k1, k2, k3, k4}
+
+	for i, sk := range tests {
```

Comment:
there's no difference between 4 tests. We could avoid the loop

Reply:
modified the test